Makefile

```diff
@@ -0,0 +1,15 @@
+all: test
```

Comment:
I am 👎 on adding makefiles, they are not cross platform and don't add much value compared to just `npm run-script`